nexus/pkg/server/filestream.go

```diff
@@ -97,9 +97,8 @@ type FileStream struct {
 }
 
 // NewFileStream creates a new filestream
-func NewFileStream(path string, settings *service.Settings, logger *observability.NexusLogger) *FileStream {
+func NewFileStream(settings *service.Settings, logger *observability.NexusLogger) *FileStream {
```

Comment:
why pull path out of the New?  it means that filestream isnt useful until we set a path

Reply:
Oh I was experimenting with moving stuff around will revert